### ISSUE TITLE
Certain playermodels cause the viewmodel to flicker in Hybrid Immersion Firstperson.

### ISSUE THREAD

**Issue by Haano137** [relaxtakenotes/cool-thirdperson-view#5]
Playermodel in question `https://steamcommunity.com/sharedfiles/filedetails/?id=3018406631`.

**Comment by relaxtakenotes** [comment]
don't know what causes this, and from my experience this happens with any mod that renders the playermodel. maybe it's related to frustum culling?

**Comment by Haano137** [comment]
> don't know what causes this, and from my experience this happens with any mod that renders the playermodel. maybe it's related to frustum culling?

Honestly, I don't know enough of the back-end side of Gmod to give a good enough comment on that. I'm just here to notify an issue I've found.

**Comment by Haano137** [comment]
Okay I've tried to look in at what you said if it also happened on other playermodel render mods. For the most part, both [Enhanced Camera 2](https://steamcommunity.com/sharedfiles/filedetails/?id=2203217139) and [Gmod Legs 3](https://steamcommunity.com/sharedfiles/filedetails/?id=112806637) seem to be working okay.

**Comment by relaxtakenotes** [comment]
> Okay I've tried to look in at what you said if it also happened on other playermodel render mods. For the most part, both [Enhanced Camera 2](https://steamcommunity.com/sharedfiles/filedetails/?id=2203217139) and [Gmod Legs 3](https://steamcommunity.com/sharedfiles/filedetails/?id=112806637) seem to be working okay.

they render the playermodel manually, by not using the drawviewer option in the calcview hook (in laymans terms: i let the game handle it, they make the game do it themselves)

**Comment by relaxtakenotes** [comment]
closing due to not being possible to fix
